src/backend/renderer/mod.rs

```diff
@@ -43,6 +43,8 @@ pub mod element;
 
 pub mod damage;
 
+pub mod test;
```

Comment:
Can you please add a new (not enabled by default) feature to this module? Something like `renderer_test`.